src/main/src/main/java/org/geoserver/security/GeoServerSecurityManager.java

```diff
@@ -2212,14 +2212,6 @@ boolean migrateFrom21() throws Exception{
         }
         
         LOGGER.info("Start security migration");
-        
-        //create required directories
-        get("security/role").dir();
-        get("security/usergroup").dir();
-        get("security/auth").dir();
-        get("security/pwpolicy").dir();
-        get("security/filter").dir();
-        get("security/masterpw").dir();
```

Comment:
This is the code that I should have been removed last year but I missed. The method does never use the file directories created there. There isn't really a way to test, because there is no expected behaviour from this. The fact that the tests for this method still pass should be enough...